test/browser_tests/cucumber/features/filterable-page.feature

```diff
@@ -0,0 +1,18 @@
+
+Feature: Filterable
+  As a user of cf.gov
+  I expect to see all results when I haven't selected a filter
+
+  Scenario: Sublanding filterable page
+  	Given I goto a sublanding filterable page
+  	When I do not select a filter on the sublanding filterable page
```

Comment:
I added "on the sublanding filterable page" and "on the browse filterable page" to make these steps unique; otherwise it complains.  This isn't really ideal -- I don't think we need this level of detail here, and it seems impractical that every step in a test will be unique across all tests. Any thoughts @sebworks ?